src/index.js

```diff
@@ -23,23 +23,17 @@ const propNames = styleKeys
   }, [])
 
 // private blacklist
-const _blacklist = [
-  'css',
-  'is',
-  'tag',
-  'extend',
-  ...propNames
-]
+const _blacklist = [ 'as', 'css', ...propNames ]
 
 const tag = React.forwardRef(({
   blacklist = [],
   ...props
 }, ref) => {
-  const Base = props.extend || props.tag || props.is || 'div'
+  const Base = props.as || 'div'
```

Comment:
The `is` prop and `extend` prop do different things. Might be good to write failing tests before editing the implementation

Reply:
yes haha, just I'm testing ideas.

How we can differenciate between `is` and `extend`? what's the mission of each prop?

Reply:
Wouldn't a `string` passed mean `is`, and `object` mean `extend`?